tests/scripts/all.sh

```diff
@@ -3849,7 +3849,8 @@ support_test_aesni() {
     # We can only grep /proc/cpuinfo on Linux, so this also checks for Linux
     (gcc -v 2>&1 | grep Target | grep -q x86_64) &&
         [[ "$HOSTTYPE" == "x86_64" && "$OSTYPE" == "linux-gnu" ]] &&
-        (lscpu | grep -qw aes)
+        (lscpu | grep -qw aes) && \
+        (grep -qF "22.04" /etc/lsb-release)
```

Comment:
I don't think we want to require a specific Ubuntu version - this shouldn't be needed IMO?

Reply:
That's a test. I will remove it if I found root cause